shared/src/models/utils/position/with-extent.ts

```diff
@@ -0,0 +1,7 @@
+import type { Size } from '../size';
+import type { Position } from './position';
+
+export interface WithExtent {
+    readonly position: Position;
+    readonly size: Size;
+}
```

Comment:
```suggestion
import type { Size } from '../size';
import type { Position } from './position';
import type { WithPosition } from './with-position'

export interface WithExtent extends WithPosition {
    readonly size: Size;
}
```

Reply:
Thank you! Fixed it in https://github.com/hpi-sam/digital-fuesim-manv/pull/663/commits/607da5d474e7c7566b9a183eb7c5966533be0ce5.